test/integration/quic_http_integration_test.cc

```diff
@@ -431,19 +432,11 @@ class QuicHttpIntegrationTestBase : public HttpIntegrationTest {
       }
     }
     for (size_t i = 0; i < concurrency_; ++i) {
-      fake_upstream_connection_ = nullptr;
-      upstream_request_ = nullptr;
       auto encoder_decoder = codec_clients[i]->startRequest(default_request_headers_);
 
       auto& request_encoder = encoder_decoder.first;
       auto response = std::move(encoder_decoder.second);
       codec_clients[i]->sendData(request_encoder, 1000, true);
-      waitForNextUpstreamRequest();
-      upstream_request_->encodeHeaders(Http::TestResponseHeaderMapImpl{{":status", "200"},
```

Comment:
do we want to set the response headers to include cookies or was that just a cut and paste rather than an intentional test?
/wait-any

Reply:
We don't need cookies for this test. The test only verifies that we can finish a request/response on each connections connected to different Envoy workers.